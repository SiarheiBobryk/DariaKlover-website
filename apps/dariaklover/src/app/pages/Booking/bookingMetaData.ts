import { PageMetaData } from '@dariaklover/types';

const bookingMetaData: PageMetaData = {
  title: 'Записаться',
  heading: 'Записаться',
  path: '/booking',
};

export default bookingMetaData;
