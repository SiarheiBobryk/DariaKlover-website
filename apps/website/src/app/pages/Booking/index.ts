export { default } from './Booking';

export { default as bookingMetaData } from './bookingMetaData';
