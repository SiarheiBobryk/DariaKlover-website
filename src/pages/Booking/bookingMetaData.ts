export interface BookingMetaData {
  title: string;
  heading: string;
  path: string;
}

const bookingMetaData: BookingMetaData = {
  title: 'Запись на консультацию',
  heading: 'Запись на консультацию (Бета)',
  path: '/booking',
};

export default bookingMetaData;
