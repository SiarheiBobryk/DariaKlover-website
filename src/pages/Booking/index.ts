export { default } from './Booking';

export { default as bookingMetaData } from './bookingMetaData';
export * from './bookingMetaData';
