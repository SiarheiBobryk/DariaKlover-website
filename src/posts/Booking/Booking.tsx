import * as React from 'react';

import Link from '@mui/material/Link';

import OpenInNewIcon from '@mui/icons-material/OpenInNew';
import Collapse from '@mui/material/Collapse';
import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';
import IconButton from '@mui/material/IconButton';
import CloseIcon from '@mui/icons-material/Close';

import { AppConfigContext, Config } from '../../providers/AppConfigProvider';
import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';

export interface BookingProps {
  heading?: string;
}

function Booking(props: BookingProps) {
  const { heading } = props;
  const {
    calendly: { label: calendlyLabel, href: calendlyHref },
  }: Config = React.useContext(AppConfigContext);
  const [openInfo, setOpenInfo] = React.useState(true);

  return (
    <>
      <Heading>{heading}</Heading>
      <Paragraph>
        –ò—Å–ø–æ–ª—å–∑—É–π—Ç–µ{' '}
        <Link sx={{ textTransform: 'lowercase' }} href={calendlyHref} target="_blank">
          {calendlyLabel} <OpenInNewIcon fontSize="small" sx={{ verticalAlign: 'middle' }} />
        </Link>{' '}
        –¥–ª—è –æ–∑–Ω–∞–∫–æ–º–ª–µ–Ω–∏—è —Å —Ä–∞—Å–ø–∏—Å–∞–Ω–∏–µ–º –∏ –¥–ª—è –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é.
      </Paragraph>
      <Collapse in={openInfo}>
        <Alert
          severity="info"
          action={
            <IconButton
              aria-label="–ó–∞–∫—Ä—ã—Ç—å"
              color="inherit"
              onClick={() => {
                setOpenInfo(false);
              }}
            >
              <CloseIcon fontSize="inherit" />
            </IconButton>
          }
          sx={{ mb: 2 }}
        >
          <AlertTitle>–ú–∞–ª–µ–Ω—å–∫–æ–µ –æ–±—ä—è–≤–ª–µ–Ω–∏–µ</AlertTitle>
          –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏ –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ Calendly, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π. –í —Ç–æ –∂–µ –≤—Ä–µ–º—è, –∫
          —Å–æ–∂–∞–ª–µ–Ω–∏—é, —ç—Ç–æ—Ç —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å
          –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π —è–∑—ã–∫ –≤—Å–µ —Å—Ç—Ä–∞–Ω–∏—á–∫–∏, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å
          100% —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é –∫–∞–ª–µ–Ω–¥–∞—Ä—è –î–∞—Ä—å–∏,
          –∫–æ—Ç–æ—Ä–∞—è –æ—á–µ–Ω—å —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
        </Alert>
      </Collapse>
    </>
  );
}

export default Booking;
