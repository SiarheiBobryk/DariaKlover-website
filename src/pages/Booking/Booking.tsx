import * as React from 'react';
import { Helmet } from 'react-helmet';
import { useSearchParams } from 'react-router-dom';

import Alert from '@mui/material/Alert';
import AlertTitle from '@mui/material/AlertTitle';

import List from '@mui/material/List';
import ListItem from '@mui/material/ListItem';
import Link from '@mui/material/Link';
import Heading from '../../components/Heading';
import Paragraph from '../../components/Paragraph';
import { AppConfigContext, Config } from '../../providers/AppConfigProvider';
import bookingMetaData from './bookingMetaData';

const calendlyResponceKeys = [
  'assigned_to',
  'event_type_uuid',
  'event_type_name',
  'event_start_time',
  'event_end_time',
  'invitee_uuid',
  'invitee_full_name',
  'invitee_email',
  'text_reminder_number',
];

function References() {
  const {
    calendly: { href: calendlyHref },
  }: Config = React.useContext(AppConfigContext);

  const [searchParams, setSearchParams] = useSearchParams();

  console.info(searchParams.get('assigned_to'));

  return (
    <>
      <Helmet>
        <title>{bookingMetaData.title}</title>
      </Helmet>
      <Heading>{bookingMetaData.heading}</Heading>
      <Paragraph>–ù–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç –¥–æ—Å—Ç—É–ø–Ω—ã —Å–ª–µ–¥—É—é—â–∏–µ –≤–∏–¥—ã –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏–π:</Paragraph>
      <List>
        <ListItem>
          <Link href={`${calendlyHref}/trial`}>–û–∑–Ω–∞–∫–æ–º–∏—Ç–µ–ª—å–Ω–∞—è –≤—Å—Ç—Ä–µ—á–∞</Link>
        </ListItem>
        <ListItem>
          <Link href={`${calendlyHref}/regular`}>Heath-–∫–æ—É—á–∏–Ω–≥ —Å–µ—Å—Å–∏—è</Link>
        </ListItem>
      </List>
      <Alert sx={{ my: 1 }} severity="warning">
        <AlertTitle>–°–∏—Å—Ç–µ–º–∞ —Ç–µ—Å—Ç–∏—Ä—É–µ—Ç—Å—è</AlertTitle>
        –û–±—Ä–∞—â–∞–µ–º –≤–∞—à–µ –≤–Ω–∏–º–∞–Ω–∏–µ, —á—Ç–æ —Å–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –≤ —Ä–µ–∂–∏–º–µ —Ç–µ—Å—Ç–∏—Ä–æ–≤–∞–Ω–∏—è –∏ –º–æ–∂–µ—Ç –±—ã—Ç—å
        –Ω–µ—Å—Ç–∞–±–∏–ª—å–Ω–∞.
      </Alert>
      <Alert sx={{ my: 1 }} severity="info">
        <AlertTitle>–†—É—Å—Å–∫–∏–π —è–∑—ã–∫</AlertTitle>
        C–∏—Å—Ç–µ–º–∞ –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π, –æ–¥–Ω–∞–∫–æ —ç—Ç–æ—Ç
        —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π
        —è–∑—ã–∫ –≤—Å–µ —ç–ª–µ–º–µ–Ω—Ç—ã –Ω–∞ —Å—Ç—Ä–∞–Ω–∏—á–∫–∞—Ö, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å 100%
        —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é,
        –∫–æ—Ç–æ—Ä–∞—è —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
      </Alert>
    </>
  );
}

export default References;
