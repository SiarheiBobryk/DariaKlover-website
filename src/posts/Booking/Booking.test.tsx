import * as React from 'react';

import { render, RenderResult } from '@testing-library/react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import { axe, toHaveNoViolations } from 'jest-axe';

import Booking from './Booking';

expect.extend(toHaveNoViolations);

describe('<Booking />', () => {
  describe('a11y', () => {
    it('should match a11y standards', async () => {
      const { container }: RenderResult = render(<Booking heading="Lorem Ipsulum" />);

      // TODO: Find out the right type here ðŸ¤”
      const results = await axe(container);
      expect(results).toHaveNoViolations();
    });
  });

  describe('snapshotting', () => {
    it('should render correctly without props', () => {
      const booking: ReactTestRendererJSON | ReactTestRendererJSON[] | null = renderer.create(<Booking />).toJSON();

      expect(booking).toMatchSnapshot();
    });

    it('should render correctly with props', () => {
      const booking: ReactTestRendererJSON | ReactTestRendererJSON[] | null = renderer
        .create(<Booking heading="Lorem Ipsulum" />)
        .toJSON();

      expect(booking).toMatchSnapshot();
    });
  });

  describe('Elements inspection', () => {
    it('should have the `h2` HTML element as heading', () => {
      render(<Booking />);

      const h2: HTMLHeadingElement | null = document.querySelector('h2');
      expect(h2).toBeInTheDocument();
    });
  });
});
