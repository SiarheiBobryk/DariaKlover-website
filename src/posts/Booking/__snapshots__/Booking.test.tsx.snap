// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Booking /> snapshotting should render correctly with props 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
  >
    Lorem Ipsulum
  </h2>,
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    –ò—Å–ø–æ–ª—å–∑—É–π—Ç–µ
     
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1yirhap-MuiTypography-root-MuiLink-root"
      href="https://calendly.com/dariaklover"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_blank"
    >
      –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏
       
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
        data-testid="OpenInNewIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
        />
      </svg>
    </a>
     
    –¥–ª—è –æ–∑–Ω–∞–∫–æ–º–ª–µ–Ω–∏—è —Å —Ä–∞—Å–ø–∏—Å–∞–Ω–∏–µ–º –∏ –¥–ª—è –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é.
  </p>,
  <div
    className="MuiCollapse-root MuiCollapse-vertical MuiCollapse-entered css-pwcg7p-MuiCollapse-root"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-12e0018-MuiPaper-root-MuiAlert-root"
          role="alert"
        >
          <div
            className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="InfoOutlinedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
              />
            </svg>
          </div>
          <div
            className="MuiAlert-message css-acap47-MuiAlert-message"
          >
            <div
              className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
            >
              –ú–∞–ª–µ–Ω—å–∫–æ–µ –æ–±—ä—è–≤–ª–µ–Ω–∏–µ
            </div>
            –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏ –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ Calendly, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π. –í —Ç–æ –∂–µ –≤—Ä–µ–º—è, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, —ç—Ç–æ—Ç —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π —è–∑—ã–∫ –≤—Å–µ —Å—Ç—Ä–∞–Ω–∏—á–∫–∏, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å 100% —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é –∫–∞–ª–µ–Ω–¥–∞—Ä—è –î–∞—Ä—å–∏, –∫–æ—Ç–æ—Ä–∞—è –æ—á–µ–Ω—å —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
          </div>
          <div
            className="MuiAlert-action css-ki1hdl-MuiAlert-action"
          >
            <button
              aria-label="–ó–∞–∫—Ä—ã—Ç—å"
              className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span
                className="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`<Booking /> snapshotting should render correctly without props 1`] = `
Array [
  <h2
    className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
  />,
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    –ò—Å–ø–æ–ª—å–∑—É–π—Ç–µ
     
    <a
      className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1yirhap-MuiTypography-root-MuiLink-root"
      href="https://calendly.com/dariaklover"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_blank"
    >
      –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏
       
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-1xzgpfl-MuiSvgIcon-root"
        data-testid="OpenInNewIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
        />
      </svg>
    </a>
     
    –¥–ª—è –æ–∑–Ω–∞–∫–æ–º–ª–µ–Ω–∏—è —Å —Ä–∞—Å–ø–∏—Å–∞–Ω–∏–µ–º –∏ –¥–ª—è –∑–∞–ø–∏—Å–∏ –Ω–∞ –∫–æ–Ω—Å—É–ª—å—Ç–∞—Ü–∏—é.
  </p>,
  <div
    className="MuiCollapse-root MuiCollapse-vertical MuiCollapse-entered css-pwcg7p-MuiCollapse-root"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardInfo MuiAlert-standard css-12e0018-MuiPaper-root-MuiAlert-root"
          role="alert"
        >
          <div
            className="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
              data-testid="InfoOutlinedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
              />
            </svg>
          </div>
          <div
            className="MuiAlert-message css-acap47-MuiAlert-message"
          >
            <div
              className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
            >
              –ú–∞–ª–µ–Ω—å–∫–æ–µ –æ–±—ä—è–≤–ª–µ–Ω–∏–µ
            </div>
            –ö–∞–ª–µ–Ω–¥–∞—Ä—å –î–∞—Ä—å–∏ –Ω–∞—Ö–æ–¥–∏—Ç—Å—è –Ω–∞ —Å—Ç–æ—Ä–æ–Ω–Ω–µ–º —Å–µ—Ä–≤–∏—Å–µ Calendly, –∫–æ—Ç–æ—Ä—ã–π –æ—á–µ–Ω—å –ø—Ä–æ—Å—Ç–æ–π –∏ –ø–æ–Ω—è—Ç–Ω—ã–π. –í —Ç–æ –∂–µ –≤—Ä–µ–º—è, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, —ç—Ç–æ—Ç —Å–µ—Ä–≤–∏—Å –Ω–µ –∏–º–µ–µ—Ç —Ä—É—Å—Å–∫–æ–≥–æ —è–∑—ã–∫–∞. –î–ª—è —Ç–µ—Ö, –∫—Ç–æ –Ω–µ –≥–æ–≤–æ—Ä–∏—Ç –ø–æ-–∞–Ω–≥–ª–∏–π—Å–∫–∏, –º—ã –æ—á–µ–Ω—å –ø–æ—Å—Ç–∞—Ä–∞–ª–∏—Å—å –ø–µ—Ä–µ–≤–µ—Å—Ç–∏ –Ω–∞ —Ä—É—Å—Å–∫–∏–π —è–∑—ã–∫ –≤—Å–µ —Å—Ç—Ä–∞–Ω–∏—á–∫–∏, –Ω–æ, –∫ —Å–æ–∂–∞–ª–µ–Ω–∏—é, –Ω–∞ –¥–∞–Ω–Ω—ã–π –º–æ–º–µ–Ω—Ç —Ç–µ—Ö–Ω–∏—á–µ—Å–∫–∏ –Ω–µ –≤–æ–∑–º–æ–∂–Ω–æ –ø—Ä–µ–¥–æ—Å—Ç–∞–≤–∏—Ç—å 100% —Ä—É—Å—Å–∫—É—é –≤–µ—Ä—Å–∏—é, –ø–æ—ç—Ç–æ–º—É –º—ã –∞–∫—Ç–∏–≤–Ω–æ —Ä–∞–±–æ—Ç–∞–µ–º –Ω–∞–¥ —Ä–∞–∑—Ä–∞–±–æ—Ç–∫–æ–π –∏–Ω—Å—Ç—Ä—É–∫—Ü–∏–∏ –ø–æ –∏—Å–ø–æ–ª—å–∑–æ–≤–∞–Ω–∏—é –∫–∞–ª–µ–Ω–¥–∞—Ä—è –î–∞—Ä—å–∏, –∫–æ—Ç–æ—Ä–∞—è –æ—á–µ–Ω—å —Å–∫–æ—Ä–æ –±—É–¥–µ—Ç –æ–ø—É–±–ª–∏–∫–æ–≤–∞–Ω–∞ –Ω–∞ —ç—Ç–æ–π —Å—Ç—Ä–∞–Ω–∏—Ü–µ. –°–ø–∞—Å–∏–±–æ –∑–∞ –≤–∞—à–µ —Ç–µ—Ä–ø–µ–Ω–∏–µ –∏ –ø–æ–Ω–∏–º–∞–Ω–∏–µ üòá
          </div>
          <div
            className="MuiAlert-action css-ki1hdl-MuiAlert-action"
          >
            <button
              aria-label="–ó–∞–∫—Ä—ã—Ç—å"
              className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
